test/ODataEndToEnd/Microsoft.OData.Service.Sample.Trippin/Web.config

```diff
@@ -63,7 +63,7 @@
     </providers>
   </entityFramework>
   <connectionStrings>
-    <add name="TrippinModel" connectionString="data source=(localdb)\MSSQLLocalDB;initial catalog=TRIPPINDB;integrated security=True;connect timeout=30;MultipleActiveResultSets=True;App=EntityFramework" providerName="System.Data.SqlClient"/>
+    <add name="TrippinModel" connectionString="data source=(localdb)\MSSQLLocalDB;initial catalog=TRIPPINE2EDB;integrated security=True;connect timeout=30;MultipleActiveResultSets=True;App=EntityFramework" providerName="System.Data.SqlClient"/>
```

Comment:
No code refer to this name, code only refer to the name which is TrippinModel, change the DB name as I created TripPin sample which use same DB, want them to run separately.